### DISCUSSION TITLE
Looking for DG developer to replace AssemblyAI with Deepgram

### DISCUSSION THREAD

**Discussion by austex78704** [deepgram/community#362] [General help] [answered]
Looking for a developer to swap out the transcription engine in a simple Vercel web app. Currently using AssemblyAI for the transcription, but a basic test with DeepGram showed much lower error rates. All code is accessible in GitHub, but I’m not a programmer, just a solopreneur consultant with basic transcription needs.

**Comment by lukeocodes** [comment, marked as answer]
Hi @austex78704 can you share the GitHub repository and I'll be able to advise

**Reply by austex78704** [reply]
https://github.com/austex78704/report-automation
